### PR TITLE
chplspell dictionary updates

### PR DESCRIPTION
**Pull request by cassella**
chplspell dictionary updates

### CODE REVIEW

**Comment by lydia-duncan** [comment]
We're officially past code freeze so I'll wait to review this until things get unfrozen.  Feel free to remind me if I don't get back to it!

File: util/devel/chplspell-dictionary.fileids.json [hunk 1]
```diff
@@ -596,6 +599,9 @@
   "eda6a798-12c0-11ed-a8be-cd81bce0efc4": [
     "compiler/dyno/lib/util/terminal.cpp"
   ],
+  "3eb6fb72-300f-11ed-b71b-d7492178b4dc": [
+    "compiler/dyno/tools/chpldoc/arg.cpp"
```
Comment (lydia-duncan) [inline util/devel/chplspell-dictionary.fileids.json:603]: Is it worth removing entries for files that have been removed?  I suspect there's some files related to the old parser implementation still listed in here that have now been removed (and eventually we'll remove the original chpldoc files, which is what prompted me to think of it)

**Review by lydia-duncan — APPROVED** [review]
Regardless of the answer to the question posed, I think this is good.  I'll probably merge Monday unless you want to act on it before I merge

File: util/devel/chplspell-dictionary.fileids.json (hunk 1 above)
Comment (cassella) [reply util/devel/chplspell-dictionary.fileids.json:553]: Good call!  There were rather a lot.

**Pull request merged**